hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/helpers/OmPrefixInfo.java

```diff
@@ -200,12 +210,25 @@ public boolean equals(Object o) {
     OmPrefixInfo that = (OmPrefixInfo) o;
     return name.equals(that.name) &&
         Objects.equals(acls, that.acls) &&
-        Objects.equals(metadata, that.metadata);
+        Objects.equals(metadata, that.metadata) &&
+        objectID == that.objectID &&
+        updateID == that.updateID;
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(name);
+    return Objects.hash(name, acls, metadata, objectID, updateID);
+  }
+
+  @Override
+  public String toString() {
+    return "OmPrefixInfo{" +
+        "name='" + name + '\'' +
+        ", acls=" + acls +
+        ", metadata=" + metadata +
+        ", objectID=" + objectID +
+        ", updateID=" + updateID +
+        '}';
```

Comment:
As the line length has been increased to 120, we could reduce the no. of lines used in this return statement

Reply:
I prefer the current format, one property per line is easier to follow and makes for cleaner diffs when making changes.